### ISSUE TITLE
How can you get the meta files: .tex > Thanks for your feedback. You can set trainer.no_partial_bn = True if batch size >= 6 in each gpu and retry it, this will not affect the accuracy. That module exists some bug with distributed training, we will fix it quickly.

### ISSUE THREAD

**Issue by SceneRec** [Sense-X/X-Temporal#17]
> Thanks for your feedback. You can set trainer.no_partial_bn = True if batch size >= 6 in each gpu and retry it, this will not affect the accuracy. That module exists some bug with distributed training, we will fix it quickly.

Thanks for the reply, but it doesn't work for my problem. 
When I set no_partial_bn = True, the log file stop at 'save_dir: checkpoint/' and with no update again. and the usage is still about 800~900M.

The changed settings in my **YAML** file are only dataset related:
`root_dir:
 train:
   meta_file: /home/renb/project/action_recognition/X-Temporal/data_labels/sthv1/train_videofolder.txt
 val:
   meta_file: /home/renb/project/action_recognition/X-Temporal/data_labels/sthv1/val_videofolder.txt
 test:
  meta_file: /home/renb/project/action_recognition/X-Temporal/data_labels/sthv1/test_videofolder.txt
`
Very confused about this.

Thanks again and waiting for you suggestion.

_Originally posted by @Amazingren in https://github.com/Sense-X/X-Temporal/issues/1#issuecomment-612012673_

**Comment by deepcs233** [comment]
Maybe need to check the `img_prefix` in your config yaml, and make sure img_prefix % num_id can get the correct file name. The default setting is image_{:05d}.jpg .
The next thing is the `video_source` in the yaml: if your training data's type is video, you may need to set it as True.
